sdk/README.md

```diff
@@ -33,13 +33,13 @@ a basic sequential pipeline.
 
 4. Compile the sample pipeline:
 
-    - `cd sdk/samples/sequential`  
-    - `dsl-compile-tekton --py ./sequential.py --output pipeline.yaml`
+    - `cd sdk/samples/parallel_and_sequential`  
```

Comment:
We should get rid of this sub-folder. If we want to distinguish between basic versus complex test cases we can add sub-directories in the future.

Reply:
I added back the sequential in case we need it for e2e testing in the future. The new example I added should cover both parallel and sequential use case.